src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  font-family: "Roboto" , sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  max-width: 100%;
+  padding: 0 50px;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.menu_list {
+  display: flex;
+}
+
+.nav_item {
+  display: flex;
+  padding-left: 20px;
+}
+
+.nav_item :first-child {
+  padding-left: 0;
+}
+
+.nav_link {
+  padding: 23px 0;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+  transition: all 0.5s;
```

Comment:
Don't set all as transition property. It's not good for the performance. Specify the exact properties
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x